src/lib/preferences.ts

```diff
@@ -109,24 +104,17 @@ export function watchSelectedThemes(cb) {
         }
         cb(getSelectedThemes());
     };
+    let subscribe = new data.Subscription<any>(call_cb);
 
-    let a = watch("goban-theme-board", call_cb);
-    let b = watch("goban-theme-black", call_cb);
-    dont_call_right_away = false;
-    let c = watch("goban-theme-white", call_cb);
-    return {
-        remove: () => {
-            a.remove();
-            b.remove();
-            c.remove();
-        }
-    };
+    subscribe.to(["goban-theme-board", "goban-theme-black"]);
```

Comment:
Not sure where the bug is, but the goban-theme watcher functionality seems to be broken currently. You should be able to update the theme and see the effects immediately on any goban on the screen